src/hls-adapter.js

```diff
@@ -211,6 +211,9 @@ export default class HlsAdapter extends BaseMediaSourceAdapter {
         Utils.Object.createPropertyPath(adapterConfig, 'abr.restrictions', abr.restrictions);
       }
     }
+    if (Utils.Object.hasPropertyPath(config, 'streaming.lowLatencyMode')) {
```

Comment:
hls default is true
https://github.com/video-dev/hls.js/blob/master/docs/API.md#lowlatencymode

do we want to override this with default false ?

Reply:
no. our default is true as well